addons/account_peppol/models/res_partner.py

```diff
@@ -90,10 +155,10 @@ def button_account_peppol_check_partner_endpoint(self):
         """
         self.ensure_one()
 
-        if not (self.peppol_eas and self.peppol_endpoint) or self.ubl_cii_format in (False, 'facturx', 'oioubl_201'):
+        if not (self.peppol_eas and self.peppol_endpoint) or self.ubl_cii_format in (False, 'facturx', 'oioubl_201', 'ciusro'):
```

Comment:
Can you use somehow the mapping to get more than the customization id like if it's available for peppol or not?